src/app/components/landingPage/APsection.tsx

```diff
@@ -33,7 +30,7 @@ const APsection: React.FC<SectionProps> = ({
           {courses.map((course, index) => (
             <li key={index} className="break-inside-avoid-column">
               <Link
-                href={`${domainName}/${currentPath}/subject/${formatCourseName(course)}`}
+                href={`/subject/${formatCourseName(course)}`}
```

Comment:
The formatCourseName function actually already exists

Reply:
That sluggifies. The ones in admin undo this sluggification. I probably need to change it later but for now its fine.